src/commands/pulls/delete-review-request.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @see: https://octokit.github.io/rest.js/#api-Pulls-deleteReviewRequest
+ * const result = await octokit.pulls.deleteReviewRequest({owner, repo, number, reviewers, team_reviewers})
+ * /repos/:owner/:repo/pulls/:number/requested_reviewers
+ */
+const flow = require("lodash.flow")
+const commonYargs = require("../../../lib/common-yargs")
+const printOutput = require("../../../lib/print-output")
+const authenticatedOctokit = require("../../../lib/octokit")
+
+/**
+ * yargs builder function.
+ *
+ * @param {import('yargs').Yargs} yargs - Instance of yargs
+ */
+const builder = yargs => {
+	const baseOptions = flow([
+		commonYargs.withToken,
+		commonYargs.withJson,
+		commonYargs.withOwner,
+		commonYargs.withRepo,
+		commonYargs.withNumber,
+		commonYargs.withReviewers,
+		commonYargs.withTeamReviewers,
+	])
+
+	return baseOptions(yargs)
+}
+
+/**
+ * Delete a review for a pull request.
+ *
+ * @param {object} argv - argv parsed and filtered by yargs
+ * @param {string} argv.token
+ * @param {string} argv.json
+ * @param {string} argv.owner
+ * @param {string} argv.repo
+ * @param {integer} argv.number
+ * @param {string} argv.number
+ * @param {string} [argv.reviewers]
+ * @param {string} [argv.team_reviewers]
+ * @throws {Error} - Throws an error if any required properties are invalid
+ */
+const handler = async ({ token, json, owner, repo, number, reviewers, team_reviewers }) => {
+
+	// Ensure that all required properties have values
+	const requiredProperties = {
+		owner,
+		repo,
+		number,
+	}
+	if (
+		Object.values(requiredProperties).some(property => !property)
+		|| (!reviewers && !team_reviewers)
```

Comment:
I think it would be better to use the `yargs.check` method to validate that at least one of `reviewers` or `team_reviewers` have been specified as options ([example from @ker-an's pull request](https://github.com/Financial-Times/github/pull/47/files#diff-912129a37fbd2f5910a03f9cb58e81d8R36)).